src/node_worker.cc

```diff
@@ -145,23 +149,51 @@ bool Worker::is_stopped() const {
   return thread_stopper_.IsStopped();
 }
 
+void Worker::SetResourceConstraints(ResourceConstraints* constraints) {
+  constraints->set_stack_limit(reinterpret_cast<uint32_t*>(stack_base_));
```

Comment:
what is your view on bringing `stack size` too under this scheme?

Reply:
@gireeshpunathil I think @bnoordhuis said somewhere that it’s hard to know for users what a reasonable stack size for a thread is, and I’d agree.